src/bundle/Resources/public/scss/_general.scss

```diff
@@ -178,4 +178,12 @@ select {
 .ez-container {
     padding: calculateRem(16px);
     background: $ibexa-white;
+
+    &--thin {
+        max-width: calculateRem(1248px);
+    }
+
+    &--create-on-the-fly {
```

Comment:
I would make it more generic, what if you will need the same styling but not for content on the fly?